samples/RefreshFlow/AngularApp/angular-cli.json

```diff
@@ -1,39 +1,41 @@
 {
   "project": {
-    "version": "1.0.0-beta.20-4",
-    "name": "openiddict-samples.refresh-flow.angularapp"
+    "version": "1.0.0-beta.26",
+    "name": "angularapp"
   },
   "apps": [
     {
       "root": "src",
-      "outDir": "./wwwroot",
+      "outDir": "dist",
       "assets": [
         "assets"
       ],
       "index": "index.html",
       "main": "main.ts",
+      "test": "test.ts",
```

Comment:
Consider also removing aspects of the angular-cli.json file that are not required. Those include: 

```
-      "environments": {                           
-        "source": "environments/environment.ts",  
-        "dev": "environments/environment.ts",     
-        "prod": "environments/environment.prod.ts"
-      }                                           
```
and
```
-  "e2e": {                               
-    "protractor": {                      
-      "config": "./protractor.conf.js"   
-    }                                    
-  },                                     
-  "test": {                              
-    "karma": {                           
-      "config": "./karma.conf.js"        
-    }                                    
-  },                                     
```